support/cas-server-support-trusted-mfa/src/main/java/org/apereo/cas/trusted/BeanNames.java

```diff
@@ -0,0 +1,37 @@
+package org.apereo.cas.trusted;
+
+import lombok.experimental.UtilityClass;
+import org.apereo.cas.CipherExecutor;
+import org.apereo.cas.trusted.web.flow.DeviceFingerprintStrategy;
+import org.apereo.cas.util.gen.RandomStringGenerator;
+import org.apereo.cas.web.support.CookieRetrievingCookieGenerator;
+
+/**
+ * A utility class containing the names of several beans trusted-mfa provides.
```

Comment:
These constants help provide some compile-time validation of dependencies. Using the constants prevents accidentally mis-spelling the bean name in a `@Qualifier` elsewhere in the code base.